internal/versions/versions.go

```diff
@@ -307,8 +307,10 @@ func (h *handler) GetLatestOsImage(cpuArchitecture string) (*models.OsImage, err
 	return latest, nil
 }
 
-func (h *handler) AddReleaseImage(releaseImageUrl, pullSecret, ocpReleaseVersion, cpuArchitecture string) (*models.ReleaseImage, error) {
+func (h *handler) AddReleaseImage(releaseImageUrl, pullSecret, ocpReleaseVersion, cpuArchitecture string, cpuArchitectures []string) (*models.ReleaseImage, error) {
```

Comment:
Are we still using cpuArchitecture?

Reply:
Yes we still are, because there may be users of `default_release_images.json` not aware of the new field - they will keep use `cpu_architecture` and not provide `cpu_architectures` at all

Reply:
Oh, AddReleaseImage is an internal API so should be fine to change it. it'll probably look clearer if we have only cpu_architectures in the signature, and the caller (clusterdeployments_controller) would just provide an array with a single arch.
Anyway, we should check in line 315 if cpu_architectures is provided. I mean, no need to use oc for fetching arch in case cpu_architectures is not empty.